### PR TITLE
Changes based on style and lint checks. (final_code_quality_15)

### PR DESCRIPTION
**Pull request by coffeedogs**
Changes based on style and lint checks. (final_code_quality_15)

### CODE REVIEW

File: src/shutdown.py [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+src/shutdown.py
+===============
+"""
+
 import os
 import Queue
 import threading
 import time
 
+import shared
```
Comment (PeterSurda) [inline src/shutdown.py]: reordering `shared` can cause problem, needs to be tested

File: src/protocol.py [hunk 2]
```diff
@@ -1,34 +1,32 @@
...
 import hashlib
 import os
 import random
 import socket
 import ssl
-from struct import pack, unpack, Struct
 import sys
 import time
 import traceback
+from binascii import hexlify
```
[the remark this answers is not in the document]
Comment (g1itch) [reply src/protocol.py]: He sorts imports like @omkar1117 wrote: `import ...` first, then `from ... import ...`.

[the remark this answers is not in the document]
Comment (PeterSurda) [reply src/protocol.py]: I see.

File: src/shutdown.py (hunk 1 above)
Comment (PeterSurda) [reply src/shutdown.py]: Tested, works ok.

File: src/plugins/notification_notify2.py [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
+===================================
+"""
+# pylint: disable=import-error
 
 import gi
-gi.require_version('Notify', '0.7')
 from gi.repository import Notify
 
+gi.require_version('Notify', '0.7')
+
```
Comment (g1itch) [inline src/plugins/notification_notify2.py]: This not works in such way. `gi.require_version()` should be called before import.

File: src/plugins/menu_qrcode.py [hunk 4]
```diff
@@ -1,18 +1,25 @@
...
 """
-
-from PyQt4 import QtGui, QtCore
-import qrcode
+# pylint: disable=import-error
 
 from pybitmessage.tr import _translate
+from PyQt4 import QtCore, QtGui
+
+import qrcode
```
Comment (g1itch) [inline src/plugins/menu_qrcode.py]: Wrong order: PyQt4 and qrcode are [third party imports](https://www.python.org/dev/peps/pep-0008/#imports).

**Pull request closed without merging**